packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -413,6 +414,16 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                                     const age = Date.now() - (snapshotCachedEntry.cacheEntryTime ??
                                         (Date.now() - 30 * 24 * 60 * 60 * 1000));
 
+                                    // In order to decrease the number of times we have to execute a snapshot refresh,
+                                    // if this is the summarizer and we have a cache entry but it is past the defaultFirstSummaryCacheExpiryTimeout,
+                                    // force the network retrieval instead as there might be a more recent snapshot available.
+                                    if (this.hostPolicy.summarizerClient &&
+                                        age > this.defaultFirstSummaryCacheExpiryTimeout) {
+                                        // eslint-disable-next-line @typescript-eslint/dot-notation
+                                        props["cacheSummarizerExpired"] = 1;
```

Comment:
Is setting to `true` better?

Reply:
nit: `summarizerCacheExpired` probably reads better.

Reply:
defaultSummarizerCacheExpryTimeout